h2/src/main/org/h2/command/Parser.java

```diff
@@ -10215,6 +10215,9 @@ private CreateLinkedTable parseCreateLinkedTable(boolean temp,
         } else if (readIf("READONLY")) {
             command.setReadOnly(true);
         }
+        if (readIf("FETCH_SIZE")) {
+            command.setFetchSize(readExpression().optimize(session));
```

Comment:
BTW, do you have a reason to accept any expression here? `readNonNegativeInt()` should be good enough, H2 doesn't support parameters in DDL operations anyway. This method will also ensure correctness of value.

Reply:
rigth